.github/workflows/build.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    name: Build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-11]
+        qt: [5.15.2]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: recursive
+
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: ${{ matrix.os }}-${{ runner.arch }}
+          max-size: 100M
+
+      - name: Cache Qt
+        id: cache-qt
+        uses: actions/cache@v1  # not v2! TODO: why though?
+        with:
+          path: ../Qt
+          key: ${{ matrix.os }}-qt-${{ matrix.qt }}
+
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v2
+        with:
+          version: ${{ matrix.qt }}
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v1.12
+        with:
+          cmake-version: '3.22.x'
+
+      - name: Build
+        run: |
+          export PATH="/usr/lib/ccache:/usr/local/opt/ccache/libexec:$PATH"
+          mkdir -p build install
+          cmake -B build -S . -DCMAKE_INSTALL_PREFIX=install
```

Comment:
Why not `/usr` as the prefix?

You can install to `install` directory with `DESTDIR=install` in the make command,

Reply:
I used the example from [BUILD.md](https://github.com/PolyMC/PolyMC/blob/068094583912797855cc7ef8045198d51eedcfac/BUILD.md#building-from-command-line)

Reply:
That's for lin-nodeps only, do you want to build with that?

Reply:
Yeah. I think the idea was just to have portable builds for PRs. That way anyone (currently only Linux and macOS) can test PRs with ease